pkg/rtc/uptrackmanager.go

```diff
@@ -56,6 +56,8 @@ type UpTrackManager struct {
 	// callbacks & handlers
 	onClose        func()
 	onTrackUpdated func(track types.MediaTrack)
+
+	simulcastTrackIds sync.Map
```

Comment:
trackIds should be removed from this on publisher close, if the same track is used to republish, this may cause issues.

Reply:
pushed a commit for this, but there is also one more way to do it:

Store both true and false values in the map, anytime a track is being reused, HandleOffer() will update the value

Reply:
@boks1971 what do you think?

Also, is there a usecase where the same RTC track is used to create two publishers? That will cause issues as well